### PR TITLE
Feature/dala 518 sandbox

### PR DESCRIPTION
**Pull request by nomad3k**
## Summary
* Routine Change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [X] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [X] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [X] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: .github/workflows/continuous-integration.yml [hunk 1]
```diff
@@ -35,11 +35,12 @@ jobs:
       - name: Install poetry
         run: pip install poetry
 
-      - name: Cache poetry packages
-        uses: actions/cache@v1
-        with:
-          path: ~/.cache/pypoetry
-          key: ${{ runner.os }}-build-cache-poetry-packages-${{ hashFiles('**/poetry.lock') }}
+      # Temporary fix to solve CI/CD issues
```
Comment (interestinginteger123) [inline .github/workflows/continuous-integration.yml:38]: Is there a date of expected fix? or just put a date of this mr to keep track of.

File: sandbox/api/FHIR/R4/index.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import { Router } from 'express';
+import response0 from './responses/empty-response.fhir';
```
Comment (interestinginteger123) [inline sandbox/api/FHIR/R4/index.js:2]: variable naming needs to mean something responseEmpty etc. Small one as it make sense what your importing just good to keep it consistent.

File: sandbox/api/FHIR/R4/index.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    response3,
+    response4
+];
+
+export default function(dependencies, config) {
+    const router = new Router();
+    
+    router.get('/CovidMedicalExemption', (req, res, next) => {
+        try {
+            const [type, id] = req.query['patient.identifier'].split('|');
```
Comment (interestinginteger123) [inline sandbox/api/FHIR/R4/index.js:20]: Makes sense debating if we need to lock down the nhs numbers or not in a dict mapping?

File: .github/workflows/continuous-integration.yml (hunk 1 above)
Comment (nomad3k) [reply .github/workflows/continuous-integration.yml:38]: No, no date yet.

File: sandbox/api/FHIR/R4/index.js (hunk 2 above)
Comment (nomad3k) [reply sandbox/api/FHIR/R4/index.js:2]: Done

**Review by interestinginteger123 — APPROVED** [review]

**Pull request merged**